config/default.json

```diff
@@ -2,7 +2,11 @@
   "bindAddress": "0.0.0.0",
   "mongo": {
     "url": "mongodb://localhost/openhim",
-    "atnaUrl": "mongodb://localhost/openhim"
+    "atnaUrl": "mongodb://localhost/openhim",
+    "connectionOptions": {},
+    "openHIMApiReadPreference": "primaryPreffered",
+    "openHIMApiWriteConcern": "1",
+    "openHIMApiReadConcern": "local"
```

Comment:
Not something you added, but just noting it here, Mongoose doesnt seem to allow for `readConcern`